index.bs

```diff
@@ -12527,6 +12529,30 @@ class Vec3 {
 
 <h2 id="changes">
 Change Log</h2>
+<h3 id="changes-2020-06-11">
```

Comment:
Previous sections had id's like "changestart<n>", with the most recent being changestart1. I didn't want to renumber everything, so I made this id just include the date of the last revision.

Let me know if I should use the same convention and renumber them.

Reply:
I use the same convention of an id with a date, for change lists in specs I'm editing